src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -51,32 +48,10 @@ public class NotificationsController extends BaseController {
     private final CaseDataService caseDataService;
     private final TransferCourtService transferCourtService;
     private final NocLetterNotificationService nocLetterNotificationService;
-    private final HwfCorrespondenceService hwfNotificationsService;
     private final UpdateFrcCorrespondenceService updateFrcCorrespondenceService;
     private final ConsentOrderNotApprovedCorresponder consentOrderNotApprovedCorresponder;
     private final ConsentOrderAvailableCorresponder consentOrderAvailableCorresponder;
     private final ConsentOrderNotApprovedSentCorresponder consentOrderNotApprovedSentCorresponder;
-    private final FinremCaseDetailsMapper finremCaseDetailsMapper;
-
-    @PostMapping(value = "/hwf-successful", consumes = APPLICATION_JSON_VALUE)
```

Comment:
I think you'll need to leave this code in temporarily due to the risk of breaking things. So you'd deploy your change in 3 stages
1. Merge the Java code - once in production the /hwf-successful endpoint would still be used
2. Merge the CCD definitions change - this would result in your new handle code being used
3. Create another PR to remove the NotificationsController code

Otherwise if you merge the deleted code in step 1 then there will be a risk that someone will trigger an event that will call /hwf-successful but the Java code will no longer be there to handle it